Libraries/Image/ImageSourcePropType.js

```diff
@@ -43,6 +43,32 @@ const ImageURISourcePropType = PropTypes.shape({
    */
   body: PropTypes.string,
   /**
+   * `cache` determines how the requests handles potentially cached
+   * responses.
+   *
+   * - `default`: Use the native platforms default strategy. `useProtocolCachePolicy` on iOS.
```

Comment:
The best documentation I can point you to is https://developer.apple.com/reference/foundation/nsurlrequest.cachepolicy. I do not know the exact details of how the iOS caching works by default, but I would assume that it will simply serve it from local cache as long as it is not expired, following HTTP standards. It is safe to assume that it would not act as `force-cache` as that is not the expected behavior according to the protocol.

Basically, this default is how things act when there is no cache policy set, ie. how all requests behave in RN today.